static/js/zamboni/mobile/buttons.js

```diff
@@ -46,33 +46,6 @@
         addons_purchased = $.map(data_purchases.split(','),
                                  function(v) { return parseInt(v, 10); });
 
-    z.startPurchase = function(manifest_url, opt) {
-        $.ajax({
-            url: opt.url,
-            type: 'post',
-            dataType: 'json',
-            /* false so that the action is considered within bounds of
-             * user interaction and does not trigger the Firefox popup blocker.
-             */
-            async: false,
-            data: {'result_type': 'json'},
-            success: function(json) {
-                $('.modal').trigger('close'); // Hide all modals
-                if (json.status == 'COMPLETED') {
-                    // Bounce back to the details page.
-                    window.location = window.location.pathname + '?status=complete';
-                } else if (json.paykey) {
-                    /* This is supposed to be a global */
-                    //dgFlow = new PAYPAL.apps.DGFlow({expType:'mini'});
-                    dgFlow = new PAYPAL.apps.DGFlow({clicked: opt.el.id});
```

Comment:
I'm not 100% sure, but the `z.startPurchase` takes a `manifest_url` which I believe was only relevant to webapps.

Also, a grep on `startPurchase` doesn't return anything in the repository.